docs/development_suite/api-console/api-design/services.md

```diff
@@ -76,29 +76,6 @@ In order to use the templates provided by Mia-Platform you can contact your refe
 At this [link](/development_suite/api-console/api-design/custom_microservice_get_started.md) you can find a more detailed guide and overview about how to create new service starting from Templates.
 :::
 
-## How to create a Microservice from a Docker Image
```

Comment:
why has this content been removed? This ought to be here since this is the documentation on how to create from docker image within the Console @akelity doesn't this go against what we said last week? 🤔

Reply:
Just a mistake, the goal was to create the tutorial without removing parts of the docs.

We are aligned, don't worry about this! :)

Reply:
great thanks, we can improve this doc regardless and cross-link to the tutorial and this piece of doc :)

Reply:
Restored this section of the docs and added a link to the tutorial.